shippable.yml

```diff
@@ -0,0 +1,108 @@
+resources:
+# Automation scripts repo
+  - name: aws_memcached_tf_repo
+    type: gitRepo
+    integration: "drship_github"
+    versionTemplate:
+      sourceName: "devops-recipes/prov_aws_memcached_terraform"
+      branch: master
+
+# AWS credentials
+  - name: aws_memcached_tf_creds
+    type: integration
+    integration: "drship_aws"
+
+# Terraform State
+  - name: aws_memcached_tf_state
+    type: state
+
+# Output of Amazon ElastiCache Memcached provisioning
+  - name: aws_memcached_tf_info
+    type: params
+    versionTemplate:
+      params:
+        SEED: "initial_version"
+
+jobs:
+# Provision Amazon ElastiCache Memcached with Terraform
+  - name: prov_aws_memcached_tf
+    type: runSh
+    steps:
+      - IN: aws_memcached_tf_repo
+        switch: off
+      - IN: aws_memcached_tf_state
+        switch: off
+      - IN: aws_memcached_tf_creds
+        switch: off
+      - TASK:
+          name: prov_aws_elasticache_memcached
+          runtime:
+            options:
+              env:
+                - aws_region: "us-east-1"
+          script:
+            - pushd $(shipctl get_resource_state "aws_memcached_tf_repo")
+            - export AWS_ACCESS_KEY_ID=$(shipctl get_integration_resource_field aws_memcached_tf_creds "accessKey")
+            - export AWS_SECRET_ACCESS_KEY=$(shipctl get_integration_resource_field aws_memcached_tf_creds "secretKey")
+            - shipctl copy_file_from_resource_state aws_memcached_tf_state terraform.tfstate .
+            - shipctl replace terraform.tfvars
+            - terraform init
+            - terraform apply -auto-approve -var-file=terraform.tfvars
+      - OUT: aws_memcached_tf_info
+        overwrite: true
+      - OUT: aws_memcached_tf_state
+    on_success:
+      script:
+        - export configuration_endpoint=$(terraform output configuration_endpoint)
```

Comment:
remove all the exports. its unnecessary we can do this in 1 step in the put res state itself